src/plots/cartesian/set_convert.js

```diff
@@ -671,24 +667,17 @@ module.exports = function setConvert(ax, fullLayout) {
                             break;
                     }
 
-                    if(
-                        (op0 === '(' ? vb > b0 : vb >= b0) &&
-                        (op1 === ')' ? vb < b1 : vb <= b1)
-                    ) return BADNUM;
+                    if(vb >= b0 && vb < b1) return BADNUM;
                 } else {
                     var vals = Lib.simpleMap(brk.values, ax.d2c).sort(Lib.sorterAsc);
                     var onOpenBound = false;
 
                     for(var j = 0; j < vals.length; j++) {
                         b0 = vals[j];
                         b1 = b0 + brk.dvalue;
-                        if(
-                            (op0 === '(' ? v > b0 : v >= b0) &&
-                            (op1 === ')' ? v < b1 : v <= b1)
-                        ) return BADNUM;
+                        if(v >= b0 && v < b1) return BADNUM;
 
-                        if(onOpenBound && op0 === '(' && v === b0) return BADNUM;
-                        onOpenBound = op1 === ')' && v === b1;
+                        if(onOpenBound && v === b0) return BADNUM;
```

Comment:
Looks like `onOpenBound` can just be 🔪 at this point?

Reply:
Good eye. Done in https://github.com/plotly/plotly.js/pull/4660/commits/c4dde701ccf07702eb1408c10387f1c540c7a6e8.